docker/7.1/Dockerfile

```diff
@@ -0,0 +1,51 @@
+FROM ubuntu:16.04
+
+ENV LANG="en_US.UTF-8" \
+    LC_ALL="en_US.UTF-8" \
+    LANGUAGE="en_US.UTF-8" \
+    TERM="xterm" \
+    DEBIAN_FRONTEND="noninteractive" \
+    SYMFONY_ALLOW_APPDEV=1 \
+    NODE_VERSION=6.11.0 \
```

Comment:
you probably don't need `SYMFONY_ALLOW_APPDEV=1` and `NODE_VERSION=6.11.0` right?